README.md

```diff
@@ -126,31 +126,31 @@ override the desktop app's default settings if desired.
 Running from Docker
 ===================
 
-The Docker image can be used to serve riot-web as a web server. The easiest way to use
+The Docker image can be used to serve element-web as a web server. The easiest way to use
 it is to use the prebuilt image:
 ```bash
-docker run -p 80:80 vectorim/riot-web
+docker run -p 80:80 vectorim/element-web
```

Comment:
This has not been done yet, see https://github.com/vector-im/element-web/issues/14930.

Reply:
Ok, i reverted this and will leave a comment on that issue.